### DISCUSSION TITLE
Is customizing appearance as hard as I think it is?

### DISCUSSION THREAD

**Discussion by spease** [lxqt/lxqt#2088] [Q&A]
I had a simple idea that seemed reasonable - set the interface to various shades of red to minimize blue light at night regardless of redshift. I set the colors in Appearance and mostly seemed to accomplish this.

However:
- The panel text gets set independently
- The window frame text is a part of openbox, and I don't see any way to set the colors in the settings...guess I have to create a new theme
- The application menu doesn't even have color overrides like the panel text. From searching, it looks like I'd also have to make a new theme?

This seems excessive for such a simple change. Am I doing this wrong?

**Comment by tsujan** [comment]
The total appearance consists of LXQt theme (panel, runner,…) + widget style + window manager. All DEs have their corresponding components, that can be styled separately. The difference is that, since LXQt doesn't (and won't) have a specific widget style or WM, it doesn't have a magic button for all three themes either.

So, yes; it's as "hard" as changing three themes.
